src/System.Private.CoreLib/src/System/Threading/LowLevelThread.cs

```diff
@@ -18,22 +19,16 @@ namespace System.Threading
 {
     internal static class LowLevelThread
```

Comment:
I didn't follow the relevance to RuntimeThread.Windows.cs, could you please elaborate?

The alternative to this class is to use LowLevelMonitor (or actually a C# LowLevelMutex without a condition) for lock purposes. LowLevelLock is significantly faster, and I don't see any reason why we should p/invoke for a lock.